### DISCUSSION TITLE
BigBlueButton 2.4 (BBB)

### DISCUSSION THREAD

**Discussion by evandro-asd** [chamilo/chamilo-lms#6084] [General]
I try to configure an server with BBB 2.4, but at try to start, delete or any action on chamilo, i receive the error page "Contate um administrador do chamilo"... The plugin version is 2.8.2 and chamilo version 1.11.16

**Comment by ywarnier** [comment]
Hi @evandro-asd, it's likely that the current development version of the plugin (2.11) is better for that, but I'm not entirely sure. We should be releasing version 1.11.18 "soon" but we already have some delay doing that.
You could *try* downloading the full plugin in version 2.11 from Github (branch 1.11.x) if you know how to do that (download the whole repository and extract the plugin/bbb/ folder, I guess) and put it in your Chamilo installation (uninstall it, copy the folder, reinstall it).
The easiest path is to wait for 1.11.18.

**Comment by evandro-asd** [comment]
Thank you very much for the quick response, I will wait for the mentioned version to carry out further tests.

**Comment by ywarnier** [comment]
Hi @evandro-asd 
This is just to ask, in case you are still around and using both Chamilo and BBB, if you had a chance to try it and if it works for you.
